code/controllers/subsystem/air.dm

```diff
@@ -122,12 +122,12 @@ var/datum/subsystem/air/SSair
 
 
 /datum/subsystem/air/proc/process_high_pressure_delta()
-	for(var/turf/T in high_pressure_delta)
+	for(var/O in high_pressure_delta)
+		var/turf/open/T = O
```

Comment:
turf/open. The definition is a leftover from me removing turf/simulated. Will fix.